dataset/diarization.py

```diff
@@ -0,0 +1,59 @@
+import os
+import json
+import tqdm
+import argparse
+import vad
+import audio
+import functools
+import transcripts
+import multiprocessing as mp
+
+
+def make_diarization_dataset(input_path: str, output_path: str, sample_rate: int, keep_intersections: bool, vad_type: str, device: str, processes: int):
+	if os.path.isdir(input_path):
+		audio_files = [os.path.join(input_path, audio_name) for audio_name in os.listdir(input_path)]
+	else:
+		audio_files = [input_path]
+
+	if vad_type == 'simple':
+		_vad = vad.PrimitiveVAD(device = device)
```

Comment:
primitive or simple? let's keep just one

Reply:
Ok, what do you prefer?